apps/web/src/components/SecurityWarningModal/SecurityWarningModal.test.tsx

```diff
@@ -0,0 +1,77 @@
+import { SecurityWarningModal, accordionItems } from "./SecurityWarningModal";
+import {
+  act,
+  dynamicModalContextMock,
+  renderInModal,
+  screen,
+  userEvent,
+  waitFor,
+} from "../../testUtils";
+
+beforeEach(() => {
+  localStorage.clear();
+});
+
+describe("<SecurityWarningModal />", () => {
+  it("renders the modal with correct title and content", async () => {
+    await renderInModal(<SecurityWarningModal />);
+
+    await waitFor(() => {
+      expect(screen.getByText("Browser Extension Security Tips")).toBeVisible();
+    });
+
+    expect(
+      screen.getByText(
+        "Please carefully review these guidelines to protect your wallet from potential security risks"
+      )
+    ).toBeVisible();
+  });
+
+  it("renders all accordion items", async () => {
+    await renderInModal(<SecurityWarningModal />);
+
+    accordionItems.forEach(({ title }) => {
```

Comment:
nit: can we have an array of titles here instead? 
If there is a fault with the `accordionItems` this test will still be green.